packages/@aws-cdk/aws-cloudwatch/lib/metric-util.ts

```diff
@@ -45,8 +46,8 @@ function metricGraphJson(metric: IMetric, yAxis?: string, id?: string) {
       }
 
       // Metric attributes that are rendered to graph options
-      if (stat.account) { options.accountId = stat.account; }
-      if (stat.region) { options.region = stat.region; }
+      if (stat.account) { options.accountId = accountIfDifferentFromStack(stat.account); }
+      if (stat.region) { options.region = regionIfDifferentFromStack(stat.region); }
```

Comment:
Why can't this be the account and region if it's the same as the stack?

Reply:
Aesthetics for one (because it would be unnecessary), and on a practical note it would cause unnecessary diffs in all currently deployed CloudWatch dasbhoards.